operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -519,11 +566,23 @@ def validateNonDynamicClusterServerHaveSameCustomChannels(self, cluster):
                  self.addError("The WebLogic configured cluster " + self.name(cluster) + " has mismatched network access point " + self.name(nap) + " in servers " + self.name(firstServer) + " and " + self.name(server) + ". All network access points in a cluster must be the same.")
                  return
 
+  def validateNonDynamicClusterLeasing(self, cluster):
+    leasingRequired = self.validateSingletonServices(cluster)
+    for server in self.env.getDomain().getServers():
+      if cluster is self.env.getClusterOrNone(server):
+        leasingRequired |= self.validateJTAMigrationPolicy(server, cluster)
+        self.validateAutoMigrationDisabled(server, cluster)
```

Comment:
Minor. Consider changing "Auto" to "WholeServer".

Reply:
updated as suggested. thanks